app/controllers/v0/backend_statuses_controller.rb

```diff
@@ -4,7 +4,7 @@ module V0
   class BackendStatusesController < ApplicationController
     skip_before_action :authenticate, only: [:show]
 
-    # GET /v0/backend_status/:service
```

Comment:
This is relatively unrelated but I didn't want to make a pr for a single comment.
https://api.va.gov/v0/backend_statuses/gibs (good)
https://api.va.gov/v0/backend_status//gibs (bad)